packages/core/saved-objects/core-saved-objects-migration-server-internal/src/actions/remove_write_block.ts

```diff
@@ -37,11 +40,12 @@ export const removeWriteBlock =
         index,
         // Don't change any existing settings
         preserve_existing: true,
-        body: {
+        settings: {
```

Comment:
not required, but I also updated these requests to not use the deprecated `body` format of the Elasticsearch-js client.